lighthouse-core/audits/dobetterweb/no-datenow.js

```diff
@@ -57,7 +57,7 @@ class NoDateNowAudit extends Audit {
     const pageHost = url.parse(artifacts.URL.finalUrl).host;
     // Filter usage from other hosts.
     const results = artifacts.DateNowUse.usage.filter(err => {
-      return url.parse(err.url).host === pageHost;
+      return url.parse(err.url || '').host === pageHost;
```

Comment:
For stacktraces that are missing `.url`, what do you think about over communicating and reporting those cases as same-domain violation? In other words:

`return err.url ? url.parse(err.url).host === pageHost : true;`